src/main/java/level4.java

```diff
@@ -1,2 +1,75 @@
-package PACKAGE_NAME;public class level4 {
+import java.util.Scanner;
+
+public class level4 {
+    public static void main(String[] args){
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        String line;
+        Task[] tasks = new Task[100];
+        boolean[] done = new boolean[100];
+        String[] type = new String[100];
+        int i = 0;
+        while (true) {
+            Scanner in = new Scanner(System.in);
+            line = in.nextLine();
+            if (line.equals("bye")) {
+                break;
+            }
+            if (line.equals("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for (int j = 0; j < i; j++) {
```

Comment:
You could choose to build an output class that helps print out text, allowing you to print out lists with a single function, without using so many lines of text